src/components/JobBuild.jsx

```diff
@@ -18,12 +18,27 @@ class JobBuild extends Component {
             </div>
         );
     }
+
+    buildResult() {
```

Comment:
Sounds good. I'm busy with client work at the moment but will make the change this week and ping you when a new commit has been pushed up. :+1: